src/Common/src/CoreLib/System/EnvironmentVariableTarget.cs

```diff
@@ -5,12 +5,12 @@
 namespace System
 {
 #if PROJECTN
```

Comment:
@jkotas do we still want this defines in coreclr?

Reply:
`#if PROJECTN` can be deleted from everywhere.

Reply:
Opened https://github.com/dotnet/coreclr/pull/26058.